generator/python/generatorConfiguration.yaml

```diff
@@ -1,25 +1,25 @@
 files:
-  api_client_builder.mustache:
+  custom/api_client_builder.java.mustache:
```

Comment:
All the extensions should be .py instead of java.
The template file names should be aligned accordingly.